plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -348,6 +358,6 @@ class ScaleBarImpl : ScaleBar, View {
     internal const val MSG_RENDER_ON_DEMAND = 0
     internal const val MSG_RENDER_CONTINUOUS = 1
     internal const val DEFAULT_MAPVIEW_WIDTH = 0F
-    internal const val INTERNAL_PADDING = 20
+    internal const val INTERNAL_PADDING = 25
```

Comment:
is this in pixels? if yes should we find a value that appropriately depends on pixelratio?

Reply:
@Chaoba please use `_DP` or `_PX` postfix so that code is better readable.